src/app/custom/translations.tsv

```diff
@@ -720,14 +720,14 @@
-"ws_post-v1-homogenize-document-type-json_title"	"Homogenize document type"	"Homogénéisation du type de document"
-"ws_post-v1-homogenize-document-type-json_description"	"Brings different forms of writing from the document type to a standardized form."	"Ramène différentes formes d’écriture du type de document à une forme standardisée."
-"ws_post-v1-homogenize-publisher-json_title"	"Homogenize publisher"	"Homogénéisation de l’éditeur"
-"ws_post-v1-homogenize-publisher-json_description"	"Brings different forms of writing from the publisher to a standardized form."	"Ramène différentes formes d’écriture d'un éditeur à une forme standardisée."
+"ws_post-v1-rnsr-json_title"	"Assigning RNSR identifier(s) to an address (Alignments)"	"Attribution d’identifiant(s) RNSR à une adresse (Alignements)"
+"ws_post-v1-rnsr-json_description"	"Assigns, using rules, one or more RNSR identifiers based on an author affiliation address and a year of publication."	"Attribue, à l’aide de règles, un ou plusieurs identifiants RNSR à partir d’une adresse d’affiliation d’auteur et d’une année de publication."
+"ws_post-v1-halauthorid-idref-json_title"	"Associate an Author IDREF with a HAL Author ID"	"Associer un IDREF Auteur à un identifiant HAL Author ID"
+"ws_post-v1-halauthorid-idref-json_description"	"Allows to make correspond the identifier IDREF authors to that of the publishers in HAL, having an identifier IDAHL authors."	"Permet de faire correspondre l’identifiant IDREF auteurs à celui des publiants dans HAL, disposant d’un identifiant IDAHL authors."
+"ws_post-v1-homogenize-document-type-json_title"	"Homogenisation of document types"	"Homogénéisation des types de documents"
+"ws_post-v1-homogenize-document-type-json_description"	"Allows the ""document type"" values in a bibliographic record from the CONDITOR database (data from HAL, Crossref, Pubmed) to be written in the same way as the ""document type"" values from the Web of Science (for 2019). "	"Permet d’homogénéiser les formes d’écriture des valeurs “types de documents” dans une notice bibliographique, issues de la base CONDITOR (données provenant de HAL, Crossref, Pubmed), à partir des valeurs “types de document” issues du Web of Science (sur l’année 2019)."
+"ws_post-v1-homogenize-publisher-json_title"	"Homogenisation of publishers"	"Homogénéisation des éditeurs"
+"ws_post-v1-homogenize-publisher-json_description"	"Used to standardise the way ""publisher"" values are written in a bibliographic record, taken from the CONDITOR database (data from HAL, Crossref, Pubmed) using ""publisher"" values taken from the Web of Science (for 2019)."	"Permet d’homogénéiser les formes d’écriture des valeurs “éditeurs” dans une notice bibliographique, issues de la base CONDITOR (données provenant de HAL, Crossref, Pubmed) à partir de valeurs “éditeurs” issues du Web of Science (sur l’année 2019)."
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality.  -->
**polish:** ‏Other English texts seem to be in american English (see line 731).

```suggestion
"ws_post-v1-homogenize-document-type-json_title"	"Homogenization of document types"	"Homogénéisation des types de documents"
"ws_post-v1-homogenize-document-type-json_description"	"Allows the ""document type"" values in a bibliographic record from the CONDITOR database (data from HAL, Crossref, Pubmed) to be written in the same way as the ""document type"" values from the Web of Science (for 2019). "	"Permet d’homogénéiser les formes d’écriture des valeurs “types de documents” dans une notice bibliographique, issues de la base CONDITOR (données provenant de HAL, Crossref, Pubmed), à partir des valeurs “types de document” issues du Web of Science (sur l’année 2019)."
"ws_post-v1-homogenize-publisher-json_title"	"Homogenization of publishers"	"Homogénéisation des éditeurs"
"ws_post-v1-homogenize-publisher-json_description"	"Used to standardize the way ""publisher"" values are written in a bibliographic record, taken from the CONDITOR database (data from HAL, Crossref, Pubmed) using ""publisher"" values taken from the Web of Science (for 2019)."	"Permet d’homogénéiser les formes d’écriture des valeurs “éditeurs” dans une notice bibliographique, issues de la base CONDITOR (données provenant de HAL, Crossref, Pubmed) à partir de valeurs “éditeurs” issues du Web of Science (sur l’année 2019)."
```

Reply:
I use UK English and i will do the same to all ws, but lodex already have a mix of US and UK English
- https://www.wordreference.com/enfr/Homogenisation
- https://www.wordreference.com/enfr/standardise